pkgs/development/python-modules/rabbitpy/default.nix

```diff
@@ -18,6 +18,11 @@ buildPythonPackage rec {
   buildInputs = [ mock nose ];
   propagatedBuildInputs = [ pamqp ];
 
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace 'pamqp>=1.6.1,<2.0' 'pamqp'
```

Comment:
Why do they have this restriction? (possible side effects)

Reply:
Perhaps open an issue asking why the restriction on `<2.0`?

Reply:
It seems the restriction of <2.0 was done before pamqp 2.0 was even released, so it seems this was made as a precaution measure. Nevertheless I did open: https://github.com/gmr/rabbitpy/issues/118

Reply:
> It seems the restriction of <2.0 was done before pamqp 2.0 was even released, so it seems this was made as a precautionary measure.

Right, and that makes think that there could be problems

>  Nevertheless I did open: gmr/rabbitpy#118

Good, at least they can know that people are using it with that version.